projects/storefrontstyles/scss/components/content/navigation/_myaccount-v2-navigation.scss

```diff
@@ -0,0 +1,49 @@
+%cx-myaccount-v2-navigation {
+  background: var(--cx-color-inverse);
+  font-family: sans-serif;
+  padding: 5px 5px 5px 5px;
+  width: 250px;
+  a {
+    font-size: var(--cx-font-medium);
+    font-weight: bold;
+    width: 250px;
+    height: 50px;
+    display: flex;
+    text-indent: 5px;
+    padding: 16px !important;
+    &:hover {
+      color: var(--cx-color-primary);
+      text-decoration: underline;
+    }
+  }
+
+  cx-navigation-ui {
+    background: var(--cx-color-inverse);
+    // Node Heading
+    span {
+      text-indent: 5px;
+      font-weight: bold;
+      display: block;
+      padding: 10px 5px 15px 5px;
+      background-color: var(--cx-color-background);
+      width: 250px;
+      height: 40px;
+      margin-top: 20px;
+    }
+
+    > nav > ul {
+      > li {
+        a {
+          border: 1px solid var(--cx-color-light);
+        }
+        margin-bottom: 20px;
+      }
+    }
+  }
+
+  .fas {
+    font-family: sans-serif, 'Font Awesome 5 Free';
+    font-size: var(--cx-font-medium);
+    display: inline-block;
+  }
```

Comment:
[QUESTION]
Could you help me understand how it all works now that the icons are prepended before the link name inside each CMS navigation link? e.g. what "CSS classes" are now required in sample data of the CMS link components for the mechanism to work?

Reply:
as per this impex https://github.tools.sap/cx-commerce/spartacussampledata/blob/release/6.x/resources/spartacussampledata/import/contentCatalogs/electronicsContentCatalog/myaccount-navigation.impex icon information is passed in the styleclasses directly . updated in code comment.will be updating the docs accordingly.

Reply:
From the impex I can understand that we set CSS classes for the CMS components e.g. `fas fa-shopping-cart`.
Would you mind pasting here a screenshot of the nested HTML structure (from the Chrome html inspection dev tools) containing the navigation html?

Why I ask: I don't have an access to your server to run it locally and check it myself. Maybe I'm missing something, I'm just confused that the final HTML might not look like how I would imagine it.

As far as I remember the Font Awesome classes are usually be applied to some "dummy" elements like `<i>`, e.g. `<i class="fas fa-shoping-cart" aria-hidden="true"></i>` which live besides the string descriptions.
 So I imagine html of our navigation link should look like:
```html
<!-- <a> contains <i> with font-awesome class , BUT the string description lives aside -->
<a href="..."><i class="fas fa-shoping-cart" aria-hidden="true"></i> Order Information</a>
```

But what I _only guess_ based on the impex and the internals of `navigation-ui.component`, I guess the html of our navigation link looks like:
```html
<!-- <a> has a font-awesome class  and inside it contains the string description  -->
<a href="..." class="fas fa-shoping-cart">Order Information</a>
```

Reply:
yes, you are right..html looks like this 
<img width="1588" alt="image" src="https://github.com/SAP/spartacus/assets/52031769/22c82b12-929c-4956-8d40-d80927b707ac">

Reply:
whether adding <cx-icon> in the navigation-ui.html based on the icon information in the styleClasses attribute will be ok?

Reply:
Thanks for pasting the UI with the HTML. It seems that it looks OK so I no longer have any concerns about it. i can see that Font Awesome simply adds a `::before` block with an icon before the string link description. Thank you for helping me understand.

So no need to change IMHO. Works nice.